content/for-developers/deploy-the-application.md

```diff
@@ -1 +1,69 @@
 # Deploy your Application
+
+1. To initiate the deployment process for your application, please proceed to the preferred SNO and access the login interface, and select `htpasswd`.
+
+![Stakater App Agility Platform Login](images/login-screen.png)
+
+2. Enter the credentials and login to the cluster. Once logged in, you should see the following screen. Select `<tenant>-dev` namespace.
+
+![Stakater App Agility Platform Landing Page](images/landing-page.png)
+![Stakater App Agility Platform Tenant View](images/tenant-view.png)
+
+3. In the upper-right corner of the screen, locate the "Copy login command" button and click on it. Afterward, kindly provide your credentials once again. Next, click on the "Display login token" option, and make sure to copy the generated login token.
+
+4. Now, use the login token you obtained to log in. 
+
+![Stakater App Agility Platform Login console](images/login-console.png)
+
+5. Now, we configure tilt-settings.json to run the application via tilt.
+
+To configure tilt-settings.json for running your application with Tilt, follow these steps:
```

Comment:
we should use a simple tilt file, without setting up this json file, we can point towards local onboarding docs for more detailed configs, 

```
allow_k8s_contexts("harcoded context") # oc config current-config
default_registry("hardcoded_registry") # oc get -n openshift-image-registry route default-route -ojsonpath='{.spec.host}'
docker_build('name','dir','')
helm('folder', namespace=namespace, values=['./tilt/values-local.yaml'])
```